packages/google_maps_flutter/google_maps_flutter_android/pigeons/messages.dart

```diff
@@ -41,6 +41,17 @@ class PlatformCircle {
   final Object json;
 }
 
+/// Pigeon equivalent of the Heatmap class.
+class PlatformHeatmap {
+  PlatformHeatmap(this.json);
+
+  /// The heatmap data, as JSON. This should only be set from
+  /// Heatmap.toJson, and the native code must interpret it according to the
+  /// internal implementation details of that method.
+  // This exists only to allow incremental migration to Pigeon.
+  final Object json;
```

Comment:
Breadcrumb for my future self if I forget, or for others looking at this PR for an example of how to add a new maps object: normally I would not allow new code to be added that did this, since this pattern in the other objects was only to allow incremental Pigeon conversion, and every instance of this is technical debt. New code should never do this, and should instead have the actual structure of a heatmap so everything is strongly typed. In this case it's fine because this PR predates my Pigeon conversion, so doing the smaller incremental change to update the PR for my changes instead of rewriting all the code is totally reasonable.